onnxscript/tools/transformers_models/phi_test.py

```diff
@@ -16,32 +15,18 @@
 import onnxscript.tools.training_helper
 import onnxscript.tools.transformers_models
 import onnxscript.tools.transformers_models.phi
-
-HAS_TRANSFORMERS = onnxscript.tools.transformers_models.has_transformers()
-
-
-def export_to_onnx(model, *input_tensors, optimize=True):
-    prog = torch.onnx.dynamo_export(model, *input_tensors)
-    model_proto = prog.model_proto
-    if optimize:
-        model_proto = onnxscript.optimizer.optimize(
-            model_proto,
-            num_iterations=2,
-            onnx_shape_inference=True,
-        )
-        model_proto = onnxscript.rewriter.rewrite(model_proto)
-        model_proto = onnx.inliner.inline_local_functions(model_proto)
-    return model_proto
+from onnxscript._internal.version_utils import has_transformers, torch_older_than
```

Comment:
```suggestion
from onnxscript._internal import version_utils
```

Reply:
The lines become longer and the linter splits the lines over multiple lines and it is difficult to read. That's why I did it that way.